<?xml version="1.0"?>
<launch>
    <arg name="camera_topic_name" default="/photoneo_center/sensor/camera_info"/>
    <arg name="output_topic_name" default="ishiyama_YOLO"/>
    <arg name="model_name" default="HV8"/>
    <arg name="f_scale" default="0.5"/>
    <arg name="cx_scale" default="0.99"/>
    <arg name="cy_scale" default="0.9"/>
    <arg name="radious" default="0.05"/>
    <arg name="timespan" default="4"/>
    
    
    <node name="Bridge" pkg="estimator" type="2D_to_3D" output="screen">
        <param name="camera_topic_name" value="$(arg camera_topic_name)"/>
        <param name="output_topic_name" value="$(arg output_topic_name)"/>
        <param name="model_name" value="$(arg model_name)"/>
        <param name="f_scale" value="$(arg f_scale)"/>
        <param name="cx_scale" value="$(arg cx_scale)"/>
        <param name="cy_scale" value="$(arg cy_scale)"/>
        <param name="radious" value="$(arg radious)"/>
        <param name="timespan" value="$(arg timespan)"/>
    </node>
</launch>